static/css/diff.less

```diff
@@ -115,4 +115,34 @@ ins {
 del {
   color: #e44028;
   text-decoration: line-through;
+}
+#back2Top {
+    width: 40px;
+    line-height: 40px;
+    overflow: hidden;
+    z-index: 999;
+    display: none;
+    cursor: pointer;
+    -moz-transform: rotate(270deg);
+    -webkit-transform: rotate(270deg);
+    -o-transform: rotate(270deg);
+    -ms-transform: rotate(270deg);
+    transform: rotate(270deg);
+    position: fixed;
+    bottom: 50px;
+    right: 0;
+    background-color: #DDD;
+    color: #555;
+    text-align: center;
+    font-size: 30px;
+    text-decoration: none;
+}
+
+#back2Top:hover {
+    background-color: #DDF;
+    color: #000;
```

Comment:
use less variable @black

Reply:
less variable @black is giving more kind of dark look and this previous is giving more kind of lighter look which is nice. Should I change it or keep it as it?

Reply:
They should be the same.

```
@black: hsl(0, 0%, 0%);
```
that is the same as hex code 000
Is the css color on hover getting overriden by something else?

`npm test` will fail when run if you don't use a variable (which will soon run under CI when #1089 is finished) so we'll need to change it.

Reply:
Thank you so much will do that!